File: src/main/java/de/aquafun3d/vanilla/listeners/RespawnListener.java
package de.aquafun3d.vanilla.listeners;

import de.aquafun3d.vanilla.utils.Utils;
import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerRespawnEvent;

public class RespawnListener implements Listener {
	@EventHandler
	public void onRespawn(PlayerRespawnEvent e){
		Player player = e.getPlayer();
		Utils.sendDeathpoint(player);
	}
}

File: src/main/java/de/aquafun3d/vanilla/listeners/RespawnListener.kt
package de.aquafun3d.vanilla.listeners

import de.aquafun3d.vanilla.Main
import de.aquafun3d.vanilla.utils.Utils
import org.bukkit.ChatColor
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerRespawnEvent


class RespawnListener: Listener {

	@EventHandler
	fun onRespawn(e: PlayerRespawnEvent) {
		val player = e.player
		Utils.sendDeathpoint(player)
	}

}